int8 ID
float32 Angle
---
int8 Success
